src/services/datasources/hooks/index.js

```diff
@@ -2,5 +2,6 @@ const updatedBy = require('./updatedBy');
 const createdBy = require('./createdBy');
 const getDatasource = require('./getDatasource');
 const restrictToDatasourceSchool = require('./restrictToDatasourceSchool');
+const protectFields = require('./protectFields');
 
-module.exports = { updatedBy, createdBy, getDatasource, restrictToDatasourceSchool };
+module.exports = { updatedBy, createdBy, getDatasource, restrictToDatasourceSchool, protectFields };
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected a line break before this closing brace.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4899819)